CMakeLists.txt

```diff
@@ -64,7 +64,8 @@ ELSE()
         FIND_PATH(_Kokkos_SETTINGS
             NAMES KokkosConfig.cmake
             PATHS ${KOKKOS_PREFIX}
-            ${TPL_Trilinos_LIBRARY_DIRS}/cmake/Kokkos)
+                  ${TPL_Trilinos_LIBRARY_DIRS}/cmake/Kokkos
+                  ${Trilinos_DIR}/../Kokkos)
```

Comment:
What is this 2nd path for?

Reply:
Good question. This was from the original pull request. Let's see if the CI is happy without it.

Reply:
We still need it in the CI.